src/main/java/com/kodilla/ecommercee/repository/UserRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+@Repository
+public interface UserRepository extends CrudRepository<User, Long> {
```

Comment:
Proponuję zmianę interfejsu, po którym dziedziczy `UserRepository` z `CrudRepository` na `JpaRepository`.
Ja używam tej drugiej właściwie z jednego powodu - z powodu różnicy w definicji metody `findAll()`:
* w `CrudRepository` ta metoda zwraca obiekt typu `Iterable<T>` i jej wynik trzeba jawnie rzutować na `List<T>`;
* w `JpaRepository` nie ma tego problemu, ponieważ typ wyniku to właśnie `List<T>`.
Ach to lenistwo :)